cmake/linux/debian/lmms-vst-full.install.in

```diff
@@ -0,0 +1 @@
+usr/lib/lmms/RemoteVstPlugin*
```

Comment:
yeah, if you'd rather I just move the completed file into the base debian directory I will